utilities_for_test.go

```diff
@@ -154,14 +154,11 @@ func NewIntegrationTestEnv(t *testing.T) IntegrationTestEnv {
 			SetInitialBalance(NewHbar(150)).
 			SetAutoRenewPeriod(time.Hour*24*81 + time.Minute*26 + time.Second*39).
 			Execute(env.Client)
-		if err != nil {
-			panic(err)
-		}
+
+		require.NoError(t, err)
 
 		receipt, err := resp.SetValidateStatus(true).GetReceipt(env.Client)
-		if err != nil {
-			panic(err)
-		}
+		require.NoError(t, err)
 
 		env.OriginalOperatorID = env.Client.GetOperatorAccountID()
```

Comment:
Isn't this called for the second time after l.148

Reply:
Removed the duplication